### DISCUSSION TITLE
Amusement Attractions and Waterparks

### DISCUSSION THREAD

**Discussion by PierceWHoward** [osmlab/name-suggestion-index#5968] [Q&A]
Hi all,

I wanted to potentially add brands for specific theme parks and water parks to make adding attractions easier and more consistent. For example, Disneyland in California would have its own brand to be applied to attractions like Space Mountain, Rise of Resistance and other rides. Could something like this be added, if so, where would the appropriate place to put them be?

Thanks.

**Comment by 1ec5** [comment]
Brands in this project are tied to how OSM’s [`brand`](https://wiki.openstreetmap.org/wiki/Key:brand) key is conventionally used in the database. Are you sure that individual rides in an amusement park are supposed to be tagged with `brand`? It’s already possible to query OSM data for attractions within a particular amusement park without additional tagging.
